nautobot/core/templates/exceptions/programming_error.html

```diff
@@ -10,7 +10,7 @@
         <code>nautobot-server migrate</code> from the command line.
     </p>
     <p>
-        <i class="mdi mdi-alert"></i> <strong>Unsupported PostgreSQL version</strong> - Ensure that PostgreSQL version 9.6 or higher is in use. You
+        <i class="mdi mdi-alert"></i> <strong>Unsupported PostgreSQL version</strong> - Ensure that PostgreSQL version 12.0 or higher is in use. You
         can check this by connecting to the database using Nautobot's credentials and issuing a query for
         <code>SELECT VERSION()</code>.
```

Comment:
Since you're already here can you update this to the following? Postgres waits for you to enter a semicolon before finishing the query so if you type the command as written it looks like nothing happened.

```suggestion
        <code>SELECT VERSION();</code>.
```

Reply:
I'm actually not sure when/if we actually encounter ProgrammingError in any real case, and maybe this block should be removed entirely now that we're enforcing the version at migration time? Might need to do some testing on what happens if I start up with a newer version of Nautobot without running `migrate` and see if I can trip this error.